designing-your-journal/en/creating-stylesheet.md

```diff
@@ -0,0 +1,69 @@
+# Create a Stylesheet
+
+In OJS, the Journal Manager, Journal Editor and Production Editor can upload a customized CSS (cascading stylesheet) file to override default CSS settings of a theme.
+
+## Identifying elements of CSS
+
+Most browsers will have several tools that can help you to inspect and test the CSS that is being used in your journal to identify the specific elements you might want to change. 
+
+For example, you can right-click on a part of a webpage (Ctrl+click on Macs), and you should see an option to “Inspect element”. In both Firefox and Chrome, this will bring up a version of “Developer tools” (named differently in each browser), which will allow you to take a closer look at the contents of a page’s HTML and CSS. 
+
+For example: 
+
+![](./assets/browser-developer-tools.png)*A screenshot of an OJS website using Firefox developer tools*
+
+The developer tool will allow you to experiment with the settings for various elements on the page and to test these modifications might look like before adding them to your CSS file where the change will be reflected on your site. Making changes within the developers tools will not alter any changes made on the website itself — just the version of the website that you are viewing in your browser. For example, you can alter the header that the site name is contained in 
+```<h1 class=“pkp_site_name”>``` to see what the site would look like if the top header had a black background. 
```

Comment:
Use single ` marks on either side when putting code in line with text. Three ` marks will put the code into a separate block.